### ISSUE TITLE
Really bad CLI performance since 1.14

### ISSUE THREAD

**Issue by Olexorus** [Querz/mcaselector#188]
**Describe the bug**
I've been using MCA Selector to trim unused chunks on my server for a while, however every version since 1.14 is much slower than 1.13.3, I haven't measured it but it feels like it's taking about 10x as long, so for now I'm stuck on 1.13.3 since at that speed the newer versions are just unusable for me. Is there any specific reason for this?

**To Reproduce**
Use both 1.13.3 and any version since 1.14 and run
```
java -jar mcaselector-[version].jar --headless --mode delete --world/region [region path] --query "LastUpdate <= 2021-03-25 AND InhabitedTime <= \"2 hours\""
```

**Expected behavior**
The performance should be similar

**Environment (please complete the following information):**
- OS: Linux Ubuntu 20.04
- Java version: openjdk 11.0.10 2021-01-19

**Comment by Querz** [comment]
With the 1.14 update I reworked how mca files are loaded, to have cleaner code for loading mca files from Minecraft 1.17.

It seems that I overlooked a crucial aspect while doing this, which is deletion and export of chunks not needing to change any chunk data. The reason why it is so slow right now is because it also writes the parsed nbt data back to the files, which is completely unnecessary. Before, it was just blindly copying the raw chunk data, which is much faster.

I fixed that in [this branch](https://github.com/Querz/mcaselector/tree/defragment).
Here's a compiled version of that branch if you want to test it:
[mcaselector-1.15.1-defragment.jar.zip](https://github.com/Querz/mcaselector/files/6284144/mcaselector-1.15.1-defragment.jar.zip)
